src/System.Collections.Specialized/tests/HybridDictionary/GetItemObjTests.cs

```diff
@@ -14,40 +10,81 @@ public class GetItemObjTests
     {
         public const int MAX_LEN = 50;          // max length of random strings
 
+        private string[] TestValues(int length)
+        {
+            if (length == 5)
+                return new string[]
+               {
+                    "",
+                    " ",
+                    "$%^#",
+                    System.DateTime.Today.ToString(),
+                    Int32.MaxValue.ToString()
+                };
+
+            string[] values = new string[length];
```

Comment:
Rather than have this different behavior based on whether length == 5 or not, could we just split this into two different methods?